Assets/Plugins/Editor/Uplift/Schemas/Extensions/Upfile.cs

```diff
@@ -242,77 +242,37 @@ private void AddOrReplaceDependency(XmlDocument document, DependencyDefinition d
 
         public virtual void LoadOverrides()
         {
-            string overrideFilePath = GetOverrideFilePath();
-
             try
             {
-                LoadOverrides(overrideFilePath);
-            }
-            catch (Exception e)
-            {
-                throw new ApplicationException("Uplift: Could not load Upfile overrides from "+overrideFilePath, e);
-            }
-        }
-
-        internal virtual string GetOverrideFilePath()
-        {
-            string homePath = (Environment.OSVersion.Platform == PlatformID.Unix ||
-                               Environment.OSVersion.Platform == PlatformID.MacOSX)
-                ? Environment.GetEnvironmentVariable("HOME")
-                : Environment.ExpandEnvironmentVariables("%HOMEDRIVE%%HOMEPATH%");
+                Repository[] overrides = GetRepositoryOverrides();
+                if (Repositories == null)
+                {
+                    Repositories = overrides;
+                }
+                else if(overrides != null)
+                {
+                    int repositoriesSize = Repositories.Length + overrides.Length;
 
-            return Path.Combine(homePath, globalOverridePath);
-        }
+                    Repository[] newRepositoryArray = new Repository[repositoriesSize];
+                    Array.Copy(Repositories, newRepositoryArray, Repositories.Length);
+                    Array.Copy(overrides, 0, newRepositoryArray, Repositories.Length, overrides.Length);
 
-        internal virtual void LoadOverrides(string path)
-        {
-            Repository[] overrides = GetOverrides(path);
-            
-            if (Repositories == null)
-            {
-                Repositories = overrides;
+                    Repositories = newRepositoryArray;
+                }
             }
-            else if(overrides != null) 
+            catch (Exception e)
```

Comment:
Maybe move this into GetRepositoryOverrides() and return an empty array there if you consider it as non failing the whole call.